fleet-tenancy/config/scope-backend/networking.k8s.io_v1_networkpolicy_deny-all.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: deny-all
+  annotations:
+    configmanagement.gke.io/namespace-selector: backend-scope
```

Comment:
We could add a comment here, something like:
```
# Actuate this resource in the namespaces with labels matched by the namespace selector
```

Reply:
added